### DISCUSSION TITLE
Custom Page Title?

### DISCUSSION THREAD

**Discussion by attzonko** [Lissy93/dashy#562] [Q&A] [answered]
Running Dashy version 2.0.4

It seems that setting `pageInfo.title` does not affect the Page Title in html, it still shows as `<title>Dashy</title>` is that intended behavior or am I doing something wrong?

**Comment by lissy93** [comment, marked as answer]
Sorry for the late reply, this should now be fixed in #663 :)

**Reply by attzonko** [reply]
Thanks for responding and fixing it. Fix confirmed.
